src/components/mainThemeProvider.tsx

```diff
@@ -18,23 +18,40 @@
 
 import * as React from 'react';
 import { ThemeProvider } from '@material-ui/styles';
-import { createMuiTheme } from '@material-ui/core';
+import { createMuiTheme, ThemeOptions } from '@material-ui/core';
 
 import AppThemeOptions from './theme';
 
+function setManualPreference(mode: 'light'|'dark') {
+  if (mode === 'light') {
+    localStorage.removeItem('theme');
```

Comment:
should we just set `light` explicitly too? that way, if we change defaults, it still remembers the last thing someone used.